editor/script/script_text_editor.cpp

```diff
@@ -2191,12 +2197,54 @@ bool ScriptTextEditor::can_drop_data_fw(const Point2 &p_point, const Variant &p_
 					String(d["type"]) == "nodes" ||
 					String(d["type"]) == "obj_property" ||
 					String(d["type"]) == "files_and_dirs")) {
+		set_drop_info_text(d);
 		return true;
 	}
 
 	return false;
 }
 
+void ScriptTextEditor::set_drop_info_text(const Dictionary &p_info) const {
+	String text;
+	String c = keycode_get_string((Key)KeyModifierMask::CMD_OR_CTRL);
+
+	static const String single_resource_text = TTR("Drag and drop to copy the file path to the script.") +
+			"\n" + vformat(TTR("Hold %s when dropping to add a const var, preloading the resource at that path."), c) +
+			"\n" + TTR("Hold Shift to switch between referencing the resource by UID/file path.") +
+			"\n" + TTR("Hold Alt when dropping to add an @export var pointing to the resource.");
+
+	static const String multiple_resources_text = TTR("Drag and drop to copy the file paths to the script.") +
```

Comment:
The strings shouldn't be static. This makes them initialized only once, making it impossible to change language.
You can move them into the conditions below, and to simplify, the string can be count-agnostic, like

```suggestion
	static const String multiple_resources_text = TTR("Drag and drop to copy the file path(s) to the script.") +
```
etc.